pkg/container/qemu_runner.go

```diff
@@ -385,18 +380,17 @@ func (bw *qemu) TerminatePod(ctx context.Context, cfg *Config) error {
 
 	clog.FromContext(ctx).Info("qemu: sending shutdown signal")
 	err := sendSSHCommand(ctx,
-		"root",
-		cfg.SSHAddress,
+		sshClient,
 		cfg,
 		nil,
 		nil,
 		nil,
-		nil,
 		false,
 		[]string{"sh", "-c", "echo s > /proc/sysrq-trigger && echo o > /proc/sysrq-trigger&"},
 	)
 	if err != nil {
-		return err
+		// in case of graceful shutdown failure, axe it with pkill
+		return syscall.Kill(-qemuPID, syscall.SIGKILL)
```

Comment:
We should check the exit code of this and return the error

Reply:
discussed offline.  Let's log it and then this is fine.

Reply:
added a warning